### DISCUSSION TITLE
How to kerning for selected text

### DISCUSSION THREAD

**Discussion by ubhoopathy** [fabricjs/fabric.js#9694] [Q&A]
I need textbox and Itext kerning for select text in fabric js,

![Screenshot 2024-02-27 161701](https://github.com/fabricjs/fabric.js/assets/149072388/1ce8da7c-e59f-416b-a37b-64daadf89535)

var canvas = new fabric.Canvas('canvas');

// Creating a text object
var text = new fabric.IText('Hello, world!', {
left: 100,
top: 100,
fontFamily: 'Arial',
fontSize: 20,
selectable: true // enabling text selection
});

canvas.add(text);

**Comment by asturur** [comment]
Kerning is a concept that is applied by the canvas and is not customizable.
What you have at your disposal is letterSpacing/charSpacing and is global for all the object.
